<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookMainWindow</class>
 <widget class="QMainWindow" name="BookMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1167</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1167</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/illust.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>← Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;background-image: url(:/background.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labe_log">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>168</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>621</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="all_Button">
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>ALL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="book_Button">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>Books/书籍</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="per_Button">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>Periodicals/期刊</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="news_Button">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>Newspaper/报刊</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>1091</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background.png);
font: 10pt &quot;黑体&quot;;</string>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>类别</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>书名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>作者</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>出版社</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>出版日期</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>金额</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>库存</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="searchbutton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/fangddajing.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="lendButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>650</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>借阅</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
